### ISSUE TITLE
Bug Report: run_scans.py throws exception when running ultraThreshold.py without --perchannel argument

### ISSUE THREAD

**Issue by AndrewLevin** [cms-gem-daq-project/vfatqc-python-scripts#169]
## Brief summary of issue
I am trying to run a trigger data threshold scan. I am using the script run_scans.py to run the script ultraThreshold.py without the --perchannel argument. The scanLog.log file contains normal-looking output and then an exception on the last line:

```
[gemuser@gem904qc8daq gemdaq]$ tail -n3 /data/bigdisk/GEM-Data-Taking/GE11_QC8/GE11-X-S-CERN-0004/threshold/vfat/trig/2018.05.29.17.39/scanLog.log 
FW scan n_triggers : 10000000
FW scan status     : 0x00000000
An exception occurred name 'vfatN' is not defined
```

### Types of issue
Bug report 

## Expected Behavior
I have never successfully run this type of scan before.

### Steps to Reproduce (for bugs)
The steps to reproduce the problem are described here: http://cmsonline.cern.ch/cms-elog/1045228

## Your Environment
* Version used: not sure
* Shell used: bash

**Comment by jsturdy** [comment]
@bdorney is this related to a previous hotfix that was implemented #167?

**Comment by bdorney** [comment]
> @bdorney is this related to a previous hotfix that was implemented #167?

Yeah, I thought this had solved the problem.  Can you confirm for me that #167 was deployed to `gem904qc8daq`?  I believe the answer is yes.  I'll try to delve deeper to workout a solution (assuming issues with build can be solved).
